app/main.js

```diff
@@ -22,6 +24,7 @@ let aboutWin = null
 let settingsWin = null
 let settings
 let isOnIndefinitePause
+let isNewVersionAvailable = null
```

Comment:
why did you created `isNewVersionAvailable`? We have global `isNewVersion` already. Do we need them both?

Reply:
I will check on it, if the global variable can be used.

Reply:
The global variable is set to true within the first second of starting the app.So, the functionality as you suggested on gitter that a user will click on 'Check for update' and then the tray menu would be pushed with the 'Download latest version' would not be achievable because as soon as the app starts it calls for a function startProcessWin which checks for the new releases and notifies us. It also sets the global.shared.isNewVersion to true. So, what will happen is, we won't be able to see the 'Check for update' in the tray menu and the tray menu would be with the 'Download latest version' functionality when the app starts.

Reply:
I think that's ok. We were planning for automatic version check anyway, right? (once a day)

There is no point in users having to manually check for new version. They will only do it if they want to do it , and if our daily check has not been faster. The "Check for update" will be available most of the time (Until there is new version and our daily checker finds it).